components/setting_item_max.jsx

```diff
@@ -93,6 +93,11 @@ export default class SettingItemMax extends React.PureComponent {
          * Text of cancel button
          */
         cancelButtonText: PropTypes.node,
+
+        /**
+         * Text of save button
+         */
+        saveButtonText: PropTypes.string,
```

Comment:
Please add test case on existing component test where `props.saveButtonText` is set.

Reply:
done